.gitlab-ci.yml

```diff
@@ -0,0 +1,67 @@
+image: openjdk:17.0.2-slim
+stages:
+  - build
+  - test
+
+variables:
+  OUT: .out
+  HISTORY: $OUT/$CI_PROJECT_NAME/history
```

Comment:
what is `$CI_PROJECT_NAME`?

Reply:
https://docs.gitlab.com/ee/ci/variables/#list-all-environment-variables